communication-et-langages.csl

```diff
@@ -0,0 +1,182 @@
+    </group>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" et-al-subsequent-min="6" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+    <layout prefix="(" suffix=")" delimiter=" ; ">
+      <group delimiter=", ">
+        <group delimiter=",">
+          <text macro="author-short"/>
+          <text macro="year-date"/>
+        </group>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="4" et-al-use-first="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="title"/>
+    </sort>
+    <layout>
+      <text macro="author" suffix=","/>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
+          <group prefix=" " delimiter=" ">
+            <text macro="title" suffix=","/>
+            <text variable="genre" suffix=","/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+          </group>
+          <text prefix=" " macro="publisher"/>
+            <date variable="issued" prefix=", " suffix=".">
+        		<date-part name="year"/>
+     		 </date>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <group prefix=" " suffix="">
+            <text macro="title" quotes="true"/>
+          </group>
+          <text value=", in" font-style="italic"/>
+          <group prefix=" " delimiter=" ">
+            <text macro="editor" suffix=" (dir.),"/>
+            <text variable="container-title" font-style="italic" suffix=","/>
+            <text variable="collection-title" suffix=","/>
+            <group suffix=".">
+              <text macro="publisher"/>
+            <date variable="issued" prefix=", " suffix=",">
+        		<date-part name="year"/>
+     		 </date>
+              <group prefix=",">
+                <text variable="page" prefix=" p. "/>
+              </group>
+            </group>
+          </group>
+        </else-if>
+        <else>
+          <group prefix=" " delimiter=" ">
+            <group suffix=",">
+              <text macro="title" quotes="true"/>
+            </group>
+            <text macro="editor"/>
+          </group>
+          <group prefix=" " suffix=".">
+            <text variable="container-title" font-style="italic"/>
+            <group prefix=", ">
+              <text variable="volume"/>
+              <text variable="issue" prefix="(" suffix=")"/>
+            </group>
+      		<date variable="issued" prefix=", " suffix=",">
```

Comment:
this suffix comma doubles with the one in the prefix below -- probably delete. (Zotero and Mendeley will not print double commas. We still like to avoid them where possible.)